docs/security/lcmaps-voms-authentication.md

```diff
@@ -60,27 +60,28 @@ Unix accounts must exist for each VO, VO role, VO group, or user you choose to s
         "%RED%<VO, VO role, VO group or user>%ENDCOLOR%" %RED%<Unix account>%ENDCOLOR%
 
 
-1.  Create Unix accounts for each VO, VO role, VO group, and user that you wish to support
-1.  Edit `/etc/osg/config.d/30-gip.ini` and specify the supported VOs per [Subcluster or ResourceEntry section](../other/configuration-with-osg-configure#subcluster-resource-entry):
-
-``` ini
-allowed_vos="VO1,VO2..."
-```
-
-The full list of VOs is located on [OIM](https://oim.opensciencegrid.org/oim/vo).
-You are not expected to support all the VOs.
-If you would like to support opportunistic usage, we recommend creating the following Unix accounts:
+1.  Create Unix accounts for each VO, VO role, VO group, and user that you wish to support.
+    The full list of VOs is located on [OIM](https://oim.opensciencegrid.org/oim/vo).
+    You are not expected to support all the VOs.
+    If you would like to support opportunistic usage, we recommend creating the following Unix accounts:
+
+    | **VO name**                                             | **Unix account(s)**                                                    |
+    |---------------------------------------------------------|------------------------------------------------------------------------|
+    | [GLOW](https://oim.opensciencegrid.org/oim/vo?id=13)    | `glow`                                                                 |
+    | [OSG](https://oim.opensciencegrid.org/oim/vo?id=30)     | `osg`                                                                  |
+    | [ATLAS](https://oim.opensciencegrid.org/oim/vo?id=35)   | `usatlas1`, `usatlas2`, `usatlas3`, `usatlas4`                         |
```

Comment:
Thanks to SOFTWARE-3183 we can probably drop `usatlas4`.

Reply:
Let's have that as a separate PR to be merged when we release SOFTWARE-3183